en/release-notes/asset-manager-upgrade-guide.md

```diff
@@ -1,344 +1,350 @@
+  `cc.loader.releaseAsset` can be replaced directly with `cc.assetManager.releaseAsset`.
 
-```js
-// before
-cc.loader.releaseAsset(texture);
+  ```js
+  // before
+  cc.loader.releaseAsset(texture);
 
-// after
-cc.assetManager.releaseAsset(texture);
-```
+  // after
+  cc.assetManager.releaseAsset(texture);
+  ```
 
-##### cc.loader.releaseRes
+- **cc.loader.releaseRes**
 
-`cc.operator.releaseRes` can be replaced directly with `cc.resources.release`.
+  `cc.operator.releaseRes` can be replaced directly with `cc.resources.release`.
 
-```js
-// before
-cc.loader.releaseRes('images/a', cc.Texture2D);
+  ```js
+  // before
+  cc.loader.releaseRes('images/a', cc.Texture2D);
 
-// after
-cc.resources.release('images/a', cc.Texture2D);
-```
+  // after
+  cc.resources.release('images/a', cc.Texture2D);
+  ```
 
-##### cc.loader.releaseAll
+- **cc.loader.releaseAll**
 
-`cc.loader.releaseAll` can be replaced directly with `cc.assetManager.releaseAll`.
+  `cc.loader.releaseAll` can be replaced directly with `cc.assetManager.releaseAll`.
 
-```js
-// before
-cc.loader.releaseAll();
+  ```js
+  // before
+  cc.loader.releaseAll();
 
-// after
-cc.assetManager.releaseAll();
-```
+  // after
+  cc.assetManager.releaseAll();
+  ```
 
 **Note**:
+
 1. For security reasons, `cc.loader.releaseResDir` does not have a corresponding implementation in `cc.assetManager`, please use `cc.assetManager.releaseAsset` or `cc.resources.release` for individual resource releases.
 
-2. Since the `cc.assetManager.releaseAsset` automatically releases dependent resources, you no longer need to explicitly call `cc.loader.getDependsRecursively`, if you need to find the dependency of the resource, please refer to the relevant API in `cc.assetManager.dependUtil`.
+2. Since the `cc.assetManager.releaseAsset` automatically releases the dependency resource, you don't need to explicitly call `cc.loader.getDependsRecursively`. If you need to find the dependency of the resource, please refer to the relevant API in `cc.assetManager.dependUtil`.
 
-3. For security reasons, the original auto-release function has been removed from `cc.assetManager` and only the auto-release set on the scene is supported, `cc.assetManager` does not implement `cc.loader.setAutoRelease`, `cc.loader.setAutoReleaseRecursively`, `cc.loader.isAutoRelease`. It is recommended that you use the new auto-release mechanism based on reference counting, see [Release Of Resources](../asset-manager/release-manager.md) documentation.
+3. For security reasons, `cc.assetManager` only supports auto-release set in the scene, the others have been removed. The `cc.loader.setAutoRelease`, `cc.loader.setAutoReleaseRecursively`, `cc.loader.isAutoRelease` APIs are not implemented in `cc.assetManager`. It is recommended that you use the new auto-release mechanism based on reference counting. Please refer to the [Release Of Resources](../asset-manager/release-manager.md) documentation for details.
```

Comment:
For security reasons, \`cc.assetManager\` only supports the Auto Release property set in the scene, and \`cc.loader.setAutoRelease\`, \`cc.loader.setAutoReleaseRecursively\`, \`cc.loader.isAutoRelease\` APIs have been removed. It is recommended that you use the new auto-release mechanism based on reference counting. Please refer to the \[Release Of Resources\]\(../asset-manager/release-manager.md\) documentation for details.